LayoutTestBase/LayoutTestBase.h

```diff
@@ -24,5 +23,35 @@ FOUNDATION_EXPORT const unsigned char LayoutTestBaseVersionString[];
 #pragma mark - Main Classes
 
 #import "LYTViewProvider.h"
+#import "LYTViewCatalogProvider.h"
 #import "LYTConfig.h"
 #import "LYTLayoutPropertyTester.h"
+
+#pragma mark - View Sizing
```

Comment:
My suggestion would be to use `#pragma mark View Sizing` instead of `#pragma mark - View Sizing`. No need to create entire sections for every one of these.